src/ui/QGCToolBar.cc

```diff
@@ -721,3 +733,27 @@ void QGCToolBar::_updateConfigurations()
         _linkSelected = false;
     }
 }
+
+/**
+ * @brief Mouse entered Message label area
+ */
+void QGCToolBar::enterMessageLabel()
+{
+    if(!_rollDownMessages)
+    {
+        QPoint p = toolBarMessageLabel->mapToGlobal(QPoint(0,0));
+        _rollDownMessages = new QGCMessageViewRollDown(MainWindow::instance(),this);
+        _rollDownMessages->setAttribute(Qt::WA_DeleteOnClose);
+        _rollDownMessages->move(mapFromGlobal(p));
+        _rollDownMessages->setMinimumSize(400,300);
```

Comment:
Can we not set a minimum size and instead set it to autoexpand to display it's contents? I'm not certain if that's possible, but it'd be nice to not cover up so much UI when trying to read messages.